### DISCUSSION TITLE
Blockchain Database

### DISCUSSION THREAD

**Discussion by vanavah-jvd** [anders94/blockchain-demo#102] [Q&A]
Hello Mr. Anders Brownworth:
I am doing an MSc in Computer Science at the University of Liverpool UK.  I am interested in blockchain technology which is one of my leactures.  I love your youtube video and your demo site which clear up a lot of things.  I would appreciate it if you can help me with this problem.  I posted it on the UOL discussion forum but no one can solve the problem.

I would like to learn how to start or connect to a blockchain database to track alcohol production and its distribution for both the private and public sectors.  This is because statistics show that 20% of alcohol sales are counterfeit.  My email address is mobilewebengineer1@gmail.com

**Comment by zamoth22** [comment]
Interesting topic i have to say, Im doing my Master of Info Tech in Australia right now and have a super interesting Blockchain class where Im learning to build my own blockchain design and simulation platform. I have done some research on the alcohol industry and how blockchain can be implemented. My blockchain prof actually created a blockchain implementation system for the wine industry here in Australia but I have to say you're on the right track, however I would suggest to focus solely on Whiskey as its the most prized alcohol type in the alcohol industry. Try to build on that niche and the rest will be much easier to do. People have been comparing Whiskey to actual BTC in some cases and more rare, since using it burns it as well.
